lib/aposResponse.js

```diff
@@ -7,10 +7,14 @@ export default async function aposResponse(req) {
   const aposUrl = new URL(url.pathname, aposHost);
   aposUrl.search = url.search;
 
-  const requestHeaders = {}
-  for (const header of req.headers) {
-    requestHeaders[header[0]] = header[1];
+  const requestHeaders = {};
+  for (const [name, value] of req.headers) {
+    const headerLower = name.toLowerCase();
+    if (!config.excludeRequestHeaders?.map(h => h.toLowerCase()).includes(headerLower)) {
```

Comment:
* Shouldn't this be an equality comparison rather than "includes" which would match a header that happens to be a substring of some unrelated header?
* Because it is on each request, it is worth making it more efficient by converting all of the excluded request headers to lowercase at startup rather than doing it over and over like this.
* So you should be able to do a simple "includes" test on `config.excludeRequestHeaders` after you've normalized it.